sample/src/commonMain/kotlin/com/swmansion/kmpmaps/sample/MapsScreen.kt

```diff
@@ -56,10 +59,38 @@ internal fun MapsScreen() {
     }
     var showAllComponents by remember { mutableStateOf(true) }
     var useGoogleMapsMapView by remember { mutableStateOf(true) }
+    var showPointGeoJson by remember { mutableStateOf(false) }
+    var showPolygonGeoJson by remember { mutableStateOf(false) }
+    var showLineGeoJson by remember { mutableStateOf(false) }
+
+    val geoJsonLayers =
+        remember(showPointGeoJson, showPolygonGeoJson, showLineGeoJson) {
+            buildList {
+                if (showPointGeoJson) {
+                    add(
+                        GeoJsonLayer(
+                            geoJson = EXAMPLE_POINT_GEO_JSON,
+                            pointStyle =
+                                PointStyle(
+                                    snippet = "Recommended food places",
+                                    infoWindowAnchorU = 0.1f,
+                                    infoWindowAnchorV = 0.7f,
+                                ),
+                        )
+                    )
+                }
+                if (showPolygonGeoJson) {
+                    add(GeoJsonLayer(geoJson = EXAMPLE_POLYGON_GEO_JSON))
+                }
+                if (showLineGeoJson) {
+                    add(GeoJsonLayer(geoJson = EXAMPLE_LINE_GEO_JSON))
+                }
+            }
+        }
 
     Column(Modifier.fillMaxHeight(), Arrangement.Bottom) {
         Map(
-            modifier = Modifier.weight(1f),
+            modifier = Modifier.weight(0.55f),
```

Comment:
The weight changes (here and in line 132) seem redundant. The original setup ensures that the `Surface` takes up as much room as it needs but no more.